### DISCUSSION TITLE
Any plans to drop the `alpha1` designation from the CRD api versions?

### DISCUSSION THREAD

**Discussion by rowanmoul** [argoproj/argo-cd#15705] [General]
Purely curious here. It's not a functional concern, but I'm just wondering why the api version is still "argoproj.io/v1alpha1"?
As far as I can tell, it's been "alpha" since this project began, despite having gone through two major versions and dozens of minor versions in between. Is the API still considered "alpha" or has the version just been mostly forgotten about? Or perhaps we're waiting for that as-yet-unplanned big breaking change to the API that justifies an API version bump?

**Comment by crenshaw-dev** [comment]
> Or perhaps we're waiting for that as-yet-unplanned big breaking change to the API that justifies an API version bump?

Yep, that. I feel like we'll probably go to v1 in 3.0.0.

**Reply by yavor-martian** [reply]
@crenshaw-dev still using `argoproj.io/v1alpha1`

**Reply by crenshaw-dev** [reply]
3.0 had a lot of stuff, but not a version bump. :-) There's an open PR to introduce [v1beta1](https://github.com/argoproj/argo-cd/pull/26032).
